packages/paste-website/src/component-examples/page-templates/ObjectsListExamples.ts

```diff
@@ -0,0 +1,291 @@
+export const FullObjectsListExample = `
+<Box
+  paddingX="space100"
+  paddingTop="space130"
+  paddingBottom="space160"
+  display="flex"
+  flexDirection="column"
+  rowGap="space130"
+>
+  <Box display="flex" columnGap="space30" justifyContent="space-between" alignItems="flex-start">
+    <Box maxWidth="size70">
+      <Box display="flex" rowGap="space30" marginBottom="space30">
+        <Breadcrumb aria-label={useUID()}>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+          <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+        </Breadcrumb>
+      </Box>
+      <Heading as="h4" variant="heading10">
+        Objects
+      </Heading>
+      <Paragraph marginBottom="space0">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi non ex risus. Aenean at ante id lectus
+        faucibus hendrerit a fringilla lorem. Pellentesque faucibus sit amet dolor vitae lacinia.
+      </Paragraph>
+    </Box>
+    <Box marginBottom="space30" marginTop="space80">
+      <ButtonGroup>
+        <Button variant="secondary">Label</Button>
+        <Button variant="primary">Label</Button>
+      </ButtonGroup>
+    </Box>
+  </Box>
+  <Box>
+    <Box display="flex" flexDirection="column" rowGap="space50" marginBottom="space90">
+      <Box display="flex" columnGap="space80" alignItems="flex-end">
+        <Box minWidth="size30" maxWidth="size80" display="flex" columnGap="space50">
+          <Box width="100%">
+            <Label htmlFor={criteria1}>Label</Label>
+            <Select id={criteria1}>1</Select>
+          </Box>
+          <Box width="100%">
+            <Label htmlFor={criteria2}>Label</Label>
+            <Select id={criteria2}>2</Select>
+          </Box>
+          <Box width="100%">
+            <Label htmlFor={criteria3}>Label</Label>
+            <Select id={criteria3}>3</Select>
+          </Box>
+        </Box>
```

Comment:
suggestion(blocking): Can you add a Grid (or something) to this so the form fields are wider, like close to what they are on the "default objects list example"?